docs/Reference/API-Methods.md

```diff
@@ -716,6 +716,8 @@ None
 
 Returns the number of hashes per second with which the node is mining. 
 
+Does not support GPU mining.
```

Comment:
Is not supported for GPU mining. (?) 

Does this sentence need to be worded the other way round?  ie, the method supports specific functionality rather than GPU mining not supporting it.

Reply:
Done